storage-bigtable/src/access_token.rs

```diff
@@ -91,41 +105,55 @@ impl AccessToken {
     }
 
     /// Call this function regularly to ensure the access token does not expire
-    pub async fn refresh(&self) {
+    pub fn try_refresh(&self) {
```

Comment:
While there is some variation in the monorepo, I would generally expect a method called `fn try_something()` to return a Result. I'd lean toward the original name, or actually propagate various errors.